src/ui/public/directives/__tests__/timepicker.js

```diff
@@ -344,13 +344,11 @@ describe('timepicker directive', function () {
 
 
       // Should update the selected option
-      var i = 0;
       _.each($scope.units, function (longUnit, shortUnit) {
         $scope.relative.unit = shortUnit;
         $scope.$digest();
 
-        expect(select.val()).to.be(i.toString());
-        i++;
+        expect(select.val().split(':')[1]).to.be(shortUnit);
```

Comment:
Not sure if this is an appropriate solution to this failing test. Would like some more manual testing against this.